job_templates/ci_job.xml.em

```diff
@@ -246,6 +251,8 @@ if "%CI_ROS2_REPOS_URL%" EQU "" (
   set "CI_ROS2_REPOS_URL=@default_repos_url"
 )
 set "CI_ARGS=%CI_ARGS% --repo-file-url %CI_ROS2_REPOS_URL%"
+if "%CI_ROS2_SUPPLEMENTAL_REPOS_URL%" NEQ "" (
+  set "CI_ARGS=%CI_ARGS% --supplemental-repo-file-url %CI_ROS2_SUPPLEMENTAL_REPOS_URL%"
```

Comment:
Inspires confidence doesn't it... I checked parentheses balancing and that one was unmatched, before merging I tested this deployed on Windows and adopting that stray paren didn't seem to wreck anything.